configure.ac

```diff
@@ -1177,9 +1177,6 @@ PKG_CHECK_MODULES(libqb, libqb >= 0.13)
 CPPFLAGS="$libqb_CFLAGS $CPPFLAGS"
 LIBS="$libqb_LIBS $LIBS"
 
-dnl libqb 0.14.0+ (2012-06)
-AC_CHECK_LIB(qb, qb_ipcs_connection_auth_set)
-
```

Comment:
That got rid of one of the extra -lqb's :)

There's another AC_CHECK_LIB() further down when checking "for acl support", I bet that'll be the other one